Instastories.py

```diff
@@ -12,29 +12,37 @@
 except ImportError as e:
     PRINT_TABLE = False
 
-#  #  #  #  #
+#####
 
 COOKIE =      {  "cookie": None,
                  "user-agent": "Instagram 10.3.2 (iPhone7,2; iPhone OS 9_3_3; en_US; en-US; scale=2.00; 750x1334) AppleWebKit/420+",
                  "cache-control": "no-cache" }
 
+EXTRA_ID = [None] # Get stories from unfollowed users by using their ID
```

Comment:
Just delete this code. I think this would also cause issue as you would iterate over none in the main logic, trying to get data out of it.